devicon.json

```diff
@@ -939,11 +957,10 @@
                 "plain-wordmark"
             ],
             "font": [
-                "plain",
-                "plain-wordmark"
+                "plain"
             ]
         },
-        "color": "#008cba",
+        "color": "#3FB6D3",
```

Comment:
Reverse the color change in this pr, since it's in #335 

```suggestion
        "color": "#008cba",
```

Reply:
I was very confused about everything, but i think i've made it 🤣 
Let me know if everything's ok!

Reply:
The `devicon.json` looks good, but now you missed to upload the flutter icons (svg) ;-) Pretty confusing, but we are on a good track 😄 Thanks for your patience! :)

Reply:
Done!
It took its time, but i think everything's ready.
I'm afraid to say this, hoping everything goes well but... let me know if theres anything i should change 😅

Reply:
To be honest: I have no clue what happend but the icons are still not commited :( Are they shown for you in this pull request? Think you just crashed github 🤣 

![image](https://user-images.githubusercontent.com/8781699/97094555-f72ec200-1655-11eb-87c4-392a649a0c3a.png)

Reply:
Interesting.. When I open your fork and the branch "flutter" i can see the icons, but the are not appearing in this pull request. Could you try to close this PR and open a new one to see what happens?

Reply:
Yes, i will, idk what's happening lol